routes/index.js

```diff
@@ -54,20 +67,10 @@ var routes = {
         // So to get a custom amount we have a plan set to 1 cent, and we supply the quantity.
         // https://support.stripe.com/questions/how-can-i-create-plans-that-dont-have-a-fixed-price
         quantity: amount,
+        currency: currency,
         stripeToken: transaction.stripeToken,
         email: transaction.email,
-        currency: currency,
-        metadata: {
-          firstname: transaction.first,
-          lastname: transaction.last,
-          country: transaction.country,
-          address: transaction.address,
-          city: transaction.city,
-          zip: transaction.code,
-          state: transaction.province,
-          locale: transaction.locale,
-          signup: transaction.signup
```

Comment:
signup was removed because it actually had no business in here.

We use is in this scope, but don't need to pass it to stripe.